### PR TITLE
[DQM] Move to python3 all scripts in DQMServices

### PR DESCRIPTION
**Pull request by jfernan2**
#### PR description:

Since CMSSW_12_0_X_2021-07-06-0300  DQM Bin By Bin tool is crashing in Jenkins tests. It looks like Jenkins test is already using python3.
This PR is urgently needed to put back the Bin By Bin tool

### CODE REVIEW

**Comment by jfernan2** [comment]
urgent

**Comment by jfernan2** [comment]
+1

**Comment by jfernan2** [comment]
please test

**Comment by jfernan2** [comment]
please test

**Comment by jfernan2** [comment]
+1

**Comment by qliphy** [comment]
+1

**Comment by smuzaffar** [comment]
@jfernan2 , looks like DQM bin by bin comparison still failing
- https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_12_0_X_2021-07-07-2300+31e7d3/43925/dqm-histo-comparison-summary.html
- https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_12_0_X_2021-07-07-2300+31e7d3/43925/dqmBinByBinLog.log
```
Running comparison:
compareHistograms.py -b /data/cmsbld/jenkins/workspace/compare-root-files-short-matrix/data/CMSSW_12_0_X_2021-07-07-2300/1000.0_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT/DQM_V0001_R000165121__Global__CMSSW_X_Y_Z__RECO.root -p /data/cmsbld/jenkins/workspace/compare-root-files-short-matrix/data/PR-31e7d3/1000.0_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT/DQM_V0001_R000165121__Global__CMSSW_X_Y_Z__RECO.root -o /data/cmsbld/jenkins/workspace/compare-root-files-short-matrix/cms-bot/dqm-comparison/dqmComparisonOutput/ -n cmssw_34401 -t 16609 -r CMSSW_12_0_X_2021-07-07-2300
Exception comparing two root files: a bytes-like object is required, not 'str'
Exception comparing two root files: a bytes-like object is required, not 'str'
```

can you please test

**Comment by jfernan2** [comment]
Thanks @smuzaffar 
I have created https://github.com/cms-sw/cmssw/pull/34411

File: DQMServices/Components/scripts/dqm-access [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```
Comment (mmusich) [inline DQMServices/Components/scripts/dqm-access:1]: was this ever test after migration, see https://cms-talk.web.cern.ch/t/dqm-access-script-is-broken-in-recent-cmssw-releases/22637

**Pull request merged**